cogs/release.py

```diff
@@ -0,0 +1,198 @@
+#un implemented
+
+from asyncio.windows_events import NULL
+import discord
+from discord import client
+from discord.ext.commands import Cog, command, has_any_role
+from discord.ext import commands
+from utils import load_config
+
+global_config = load_config()
+config = global_config
+
+# TWITTER KEYS (...would go here)
+consumer_key = None
+consumer_secret = None
+access_token = None
+access_token_secret = None
+
+#colors
+tf2_color = 0xB98A2E
+art_color = 0xde1f1f
+other_color = 0x868686
+
+class twitter(Cog):
+
+    @commands.command()
+    @has_any_role('Staff', 'Server Mods', 'Senior Staff', 'Fub')
+    async def release(self, ctx):
+
+        #delete message in channel it was sent
+        await ctx.message.delete()
+        #sends message as dm
+        msg = await ctx.author.send("Welcome to TF2Maps release submission. Type cancel at anytime to stop the submission.")
+        
+        #check to make sure the link is sent in dms
+        def check(message):
+            return message.author == ctx.author and message.channel == msg.channel
+
+        step = 0  
+        submissioncategory = ''
+        submissionlink = ''
+        canceled = False
+        img = ''
+        #test number
+        while(step < 10): 
+            #get submission category    
+            if(step == 0):
```

Comment:
we can refactor this part. 

We dont need to have this step  stuff. we can have this just run serially unless there is more complicated user flow im not seeing here.

Also this seems like a good thing to try to use the interaction api with